WORKSPACE

```diff
@@ -2,16 +2,24 @@ workspace(name = "grpc_ecosystem_grpc_gateway")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+# master as of 2018-11-09. This should be updated when a new release is tagged.
```

Comment:
I don't have a great understanding of this, but does this update any transative dependencies? We need to keep this file in sync with `Gopkg.toml`.

Reply:
It does, thanks for pointing that out. I've updated it and rerun `dep ensure`, is there anything else to do?

Reply:
I guess we still want to make sure the tests pass and everything with the new versions, but I assume Bazel will do that for us as well? Just commit it and we shall see :).